keps/sig-cloud-provider/0003-testgrid-e2e.md

```diff
@@ -0,0 +1,270 @@
+---
+kep-number: 003
+title: Reporting Conformance Test Results to Testgrid
+authors:
+  - "@andrewsykim"
+owning-sig: sig-cloud-provider
+participating-sigs:
+  - sig-testing
+  - sig-release
+  - sig-aws
+  - sig-azure
+  - sig-gcp
+  - sig-ibmcloud
+  - sig-openstack
+  - sig-vmware
+reviewers:
+  - TBD
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2018-06-06
+last-updated: 2018-06-06
+status: provisional
+
+---
+
+# Reporting Conformance Test Results to Testgrid
+
+## Table of Contents
+
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+    * [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+    * [Risks and Mitigations](#risks-and-mitigations)
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+
+## Summary
+
+This is a KEP outlining the motivation behind why cloud providers should frequently upload E2E conformance test results to [Testgrid](https://github.com/kubernetes/test-infra/tree/master/testgrid) and how a cloud provider can go about doing this.
+
+## Motivation
+
+The primary motivation behind collecting conformance test results from various cloud providers on a regular basis is to inform sig-release of any critical bugs. It's important to collect results from various cloud providers to increase coverage and inform other cloud providers of bugs they may be impacted by.
+
+### Goals
+
+* All SIGs / Subprojects owners representing a cloud provider is aware of the importance of frequently uploading conformance test results to testgrid
+* There is a clear and detailed process in place for any cloud provider to upload conformance test results to Testgrid.
+
+### Non-Goals
+
+* Test coverage - increasing test coverage is outside the scope of this KEP
```

Comment:
While I don't think we want to address general test coverage issues, I wonder if we should either address the number of if not "gce" skip test or at minimum mention that conformance tests should be covered. It seems like if a sig is going to own getting rid of cloud-provider skip tests it should be sig-cloud-provider. Whats more any test which is skipped according to cloud provider is probably an important test to run. The functionality under test would seem to be related to something about cloud-provider.

Reply:
@mml suggested that we might at least want to track how many tests were skipped based on cloud provider in testgrid.

Reply:
This is not the final version of the proposal so we can refine some of these details in the future when we have more context.